src/foam/nanos/notification/EmailSetting.js

```diff
@@ -91,13 +91,12 @@ foam.CLASS({
           if ( config != null ) {
             notification.getEmailArgs().put("link", config.getUrl());
           }
-        }    
+        }
 
         try {
           if ( foam.util.SafetyUtil.isEmpty(notification.getEmailName()) ) {
-            message.setSubject(notification.getTemplate());
             message.setBody(notification.getBody());
-            EmailsUtility.sendEmailFromTemplate(x, null, message, null, null);
+            EmailsUtility.sendEmailFromTemplate(x, null, message, "NOCTemplate", null);
```

Comment:
Was this for testing?

Reply:
added 'NOCTemplate' https://github.com/nanoPayinc/NANOPAY/pull/11878/files
to fix blank email notification, it happened on 'NOC' type notification that does not have a template.
so just added NOCTemplate - for no template cases

Reply:
I'm surprised to see we've lost our 'NOC' notification template.  Why not just create a notification template with the an email name? 

notification.template: 'NOC'
notification.emailName: 'NOC'

and then the corresponding emailTemplate, with id 'NOC'.

Reply:
I believe we do have "NOC" template which located in admin > notificationTemplates. that is the one it use.

Reply:
Yes, so that needs to be updated with a valid emailName - which is the emailTemplate.